frontend/src/cards/TableCard.tsx

```diff
@@ -39,6 +40,11 @@ export interface TableCardProps {
   variableConfig: VariableConfig;
 }
 
+export const NeverShowProperties = [
```

Comment:
in general we've been using `SCREAMING_SNAKE_CASE` for constants; not sure if it's worth changing for consistency

Reply:
maybe worth adding a comment here too so we know later why we added this box of never-show properties

Reply:
For sure